wordle-web/scripts/stopwatch.ts

```diff
@@ -0,0 +1,25 @@
+
+
+
```

Comment:
brilliant way to make a stop watch.  It is a hybrid version of the two ways that I was working with.  Works much better than mine.